main.tf

```diff
@@ -3,95 +3,94 @@ locals {
 
   sku {
-    name = "standard"
+    name = "${var.sku}"
   }
 
   tenant_id = "${var.tenant_id}"
 
-  access_policy {
-    tenant_id = "${var.tenant_id}"
-    object_id = "${var.object_id}"
+  enabled_for_disk_encryption     = true
+  enabled_for_deployment          = true
+  enabled_for_template_deployment = true
 
-    certificate_permissions = [
-      "create",
-      "delete",
-      "deleteissuers",
-      "get",
-      "getissuers",
-      "import",
-      "list",
-      "listissuers",
-      "setissuers",
-      "update",
-      "managecontacts",
-      "manageissuers",
-    ]
+  tags = "${var.common_tags}"
+}
 
-    key_permissions = [
-      "create",
-      "list",
-      "get",
-      "delete",
-      "update",
-      "import",
-      "backup",
-      "restore",
-    ]
+resource "azurerm_key_vault_access_policy" "creator_access_policy" {
+  key_vault_id = "${azurerm_key_vault.kv.id}"
 
-    secret_permissions = [
-      "set",
-      "list",
-      "get",
-      "delete",
-    ]
-  }
+  object_id = "${var.object_id}"
+  tenant_id = "${var.tenant_id}"
 
-  access_policy {
-    object_id = "${var.product_group_object_id}"
-    tenant_id = "${var.tenant_id}"
+  certificate_permissions = [
+    "create",
+    "delete",
+    "deleteissuers",
+    "get",
+    "getissuers",
+    "import",
+    "list",
+    "listissuers",
+    "setissuers",
+    "update",
+    "managecontacts",
+    "manageissuers",
+  ]
 
-    key_permissions = [
-      "list",
-      "update",
-      "create",
-      "import",
-      "delete"
-    ]
-    certificate_permissions = [
-      "list",
-      "update",
-      "create",
-      "import",
-      "delete",
-      "managecontacts",
-      "manageissuers",
-      "getissuers",
-      "listissuers",
-      "setissuers",
-      "deleteissuers"
-    ]
-    secret_permissions = [
-      "list",
-      "set",
-      "delete"
-    ]
-  }
+  key_permissions = [
+    "create",
+    "list",
+    "get",
+    "delete",
+    "update",
+    "import",
+    "backup",
+    "restore",
+  ]
 
-  enabled_for_disk_encryption = true
-  enabled_for_deployment = true
-  enabled_for_template_deployment = true
+  secret_permissions = [
+    "set",
+    "list",
+    "get",
+    "delete",
+  ]
+}
 
-  tags = "${merge(var.common_tags,
-    map(
-      "environment", var.env,
-      "product", var.product
-    )
-  )}"
+resource "azurerm_key_vault_access_policy" "product_team_access_policy" {
+  key_vault_id = "${azurerm_key_vault.kv.id}"
 
-}
+  object_id = "${var.product_group_object_id}"
+  tenant_id = "${var.tenant_id}"
 
+  key_permissions = [
+    "list",
+    "update",
+    "create",
+    "import",
+    "delete",
+  ]
+
+  certificate_permissions = [
+    "list",
+    "update",
+    "create",
+    "import",
+    "delete",
+    "managecontacts",
+    "manageissuers",
+    "getissuers",
+    "listissuers",
+    "setissuers",
+    "deleteissuers",
+  ]
+
+  secret_permissions = [
+    "list",
+    "set",
+    "delete",
```

Comment:
do we need 'get' here?

Reply:
no the `product` team have never had direct get access
they can write update and delete but can't see secrets in there